plugins/broker-mqtt/src/tests/__snapshots__/createControllerFromTopics.spec.ts.snap

```diff
@@ -0,0 +1,51 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Testing the creation of topic subscription in controller should create a controller with topics if resources are found 1`] = `
+[
+  {
+    "code": "import { Controller } from "@nestjs/common";
+import { Ctx, EventPattern, MqttContext, Payload } from "@nestjs/microservices";
+
+@Controller("mqtt-controller")
+export class MqttController {
+  @EventPattern(test-topic-name)
+  async onTestTopicName(
+    @Payload()
+    value: string | Record<string, any> | null,
+    context: MqttContext
+  ): Promise<void> {
+    const message = context.getPacket();
```

Comment:
Not mandatory but it would make sense to be consistent with the integration concepts.  My suggestion is to replace the message var with originalPacket

```suggestion
    const originalPacket = context.getPacket();
```